lib/index.js

```diff
@@ -108,26 +124,50 @@ FSTree.prototype = {
     }
   },
 
-  get _changes() {
-    return this._parent ? this._parent._changes : this.__changes;
+  get exclude() {
+    return this._parent ? this._parent._exclude : this._exclude;
   },
 
-  get _hasEntries() {
-    return this._parent ? this._parent._hasEntries : this.__hasEntries;
+  set exclude(value) {
+    if (this._parent) {
+      this._parent._exclude = value;
```

Comment:
as above, we want to defer to the parent's setter, not update the parent's raw value